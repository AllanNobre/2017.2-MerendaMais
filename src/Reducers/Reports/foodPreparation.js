const foodPreparation = [
  { label: 'Higienização de frutas e verduras é realizada de maneira adequada', markedYes: false, markedNo: false, status: false, key: 0 },
  { label: 'A água sanitária utilizada é própria para alimentos', markedYes: false, markedNo: false, status: false, key: 1 },
  { label: 'O descongelamento dos alimentos é feito na geladeira', markedYes: false, markedNo: false, status: false, key: 2 },
  { label: 'Uso de água potável no preparo de gelo e alimentos', markedYes: false, markedNo: false, status: false, key: 3 },
  { label: 'Os alimentos preparados, que serão congelados ou refrigerados, são adequadamente acondicionados e identificados com: nome da preparação, data de preparo e prazo de validade após a abertura ou retirada da embalagem original', markedYes: false, markedNo: false, status: false, key: 4 },
  { label: 'Caso as refeições aconteçam em sala de aula, a escola possui carrinho para transportar os alimentos?', markedYes: false, markedNo: false, status: false, key: 5 },
  { label: 'Os carrinhos estão em boas condições e em quantidades suficientes', markedYes: false, markedNo: false, status: false, key: 6 },
];

export default foodPreparation;
