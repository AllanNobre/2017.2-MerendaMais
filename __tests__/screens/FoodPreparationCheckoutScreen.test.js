import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// imported as a connected component!
import FoodPreparationCheckoutContainer from '../../src/Containers/FoodPreparationCheckoutContainer';
import foodPreparation from '../../src/Reducers/Reports/foodPreparation';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {
  report: {
    foodPreparation,
    foodPreparationObservation: '',
  },
};

const store = mockStore(initialState);

describe('Testing FoodQualityCheckout Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <FoodPreparationCheckoutContainer />,
      { context: { store } },
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
