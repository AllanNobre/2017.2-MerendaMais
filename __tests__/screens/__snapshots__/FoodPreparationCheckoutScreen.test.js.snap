// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FoodQualityCheckout Screen renders as expected 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <Header
    backButton={true}
    subTitle="Preparação e Distribuição"
    title="Relatório"
  />
  <KeyboardAvoidingView
    behavior="padding"
    keyboardVerticalOffset={0}
    style={
      Object {
        "flex": 6,
        "flexDirection": "column",
        "marginBottom": 9,
      }
    }
  >
    <ScrollViewMock>
      <View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "fontWeight": "bold",
                "paddingLeft": 10,
                "paddingTop": 5,
              }
            }
          >
            Sim
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "fontWeight": "bold",
                "paddingLeft": 20,
                "paddingTop": 5,
              }
            }
          >
            Nao
          </Text>
        </View>
        <View
          key="0"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Higienização de frutas e verduras é realizada de maneira adequada
          </Text>
        </View>
        <View
          key="1"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            A água sanitária utilizada é própria para alimentos
          </Text>
        </View>
        <View
          key="2"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            O descongelamento dos alimentos é feito na geladeira
          </Text>
        </View>
        <View
          key="3"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Uso de água potável no preparo de gelo e alimentos
          </Text>
        </View>
        <View
          key="4"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Os alimentos preparados, que serão congelados ou refrigerados, são adequadamente acondicionados e identificados com: nome da preparação, data de preparo e prazo de validade após a abertura ou retirada da embalagem original
          </Text>
        </View>
        <View
          key="5"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Caso as refeições aconteçam em sala de aula, a escola possui carrinho para transportar os alimentos?
          </Text>
        </View>
        <View
          key="6"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Os carrinhos estão em boas condições e em quantidades suficientes
          </Text>
        </View>
      </View>
      <View
        behavior="padding"
      >
        <View
          style={
            Object {
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 30,
            }
          }
        >
          <TextInput
            multiline={true}
            onChangeText={[Function]}
            placeholder="Observações (opcional)"
            style={
              Object {
                "borderRadius": 12,
                "borderWidth": 1,
                "fontSize": 37.5,
                "height": 333.5,
                "paddingLeft": 10,
                "paddingTop": 10,
                "textAlignVertical": "top",
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <TouchableOpacity
        activeOpacity={0.2}
        key="setObservation"
        onPress={[Function]}
        style={
          Object {
            "backgroundColor": "#FF9500",
            "borderRadius": 7,
            "borderWidth": 1,
            "justifyContent": "flex-end",
            "marginBottom": 20,
            "marginHorizontal": 15,
            "marginTop": 30,
            "paddingVertical": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFF",
              "textAlign": "center",
            }
          }
        >
          Concluir
        </Text>
      </TouchableOpacity>
    </ScrollViewMock>
  </KeyboardAvoidingView>
</View>
`;
